### DISCUSSION TITLE
serializers | Additional keyword arguments -

### DISCUSSION THREAD

**Discussion by LiorA1** [encode/django-rest-framework#8044] [General]
Hi,
I'm reading the 'Additional keyword arguments' in the ModelSerializer section .
Its explain about 'extra_kwargs', but doesnt state what are the options for it..

Only 'write_only' and 'read_only' are the options for it ?
If not, what are the possible options ?

**Comment by erwazzzzzzz2** [comment]
I was looking for this also, not sure why no answers , but here is what I found.
This issue seems to be the same as #8877 
The suggested documentation is here https://www.django-rest-framework.org/api-guide/fields/
